src/main/java/no/unit/nva/database/UserDb.java

```diff
@@ -35,6 +35,8 @@
     private String username;
```

Comment:
Add a space here to separate fields from the constants

Reply:
Between primaryHashKey and username?